src/schemas/cluster_workload_info_schema.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env pytohn3
+"""Contains the marshmallow schema for cluster info entity to facilitate quick serialization."""
+from config import ma
+from models.cluster_workload_info_model import ClusterWorkloadInfoModel
+
+
+class ClusterWorkloadInfoSchema(ma.SQLAlchemySchema):
+    """Define a serialization schema for cluster workload info."""
+
+    class Meta:
```

Comment:
class inside class - Is this suggested practice for SQLAlchemy?

Reply:
This is marshmallow, and yes Metaclasses are for configuration of object: https://flask-marshmallow.readthedocs.io/en/latest/